packages/css/src/plugins/minify.tsx

```diff
@@ -0,0 +1,20 @@
+import cssnano from 'cssnano-preset-default';
+
+/**
+ * PostCSS `cssnano` plugin to minify CSS.
+ * It deliberately excludes some plugins so it is forced to be synchronous.
+ */
+export const minify = () => {
```

Comment:
nit: Shouldn't this be a default export? 🤔

Reply:
ah personal preference - if we use named exports/imports they're always named as the original author expected :)